lib/service/cfg.go

```diff
@@ -45,6 +45,9 @@ type Config struct {
 	// (in case of auth server backed by BoltDB) or local state, e.g. keys
 	DataDir string
 
+	//DisableTLS is enable if we don't want self signed certs
+	DisableTLS bool
```

Comment:
This section is for global teleport settings (like data dir). `DisableTLS` needs to exist in `ProxyConfig` instead, because it only applies if Teleport is running with `--roles=proxy` flag.